src/main/python/wheel_axle/bdist_axle/_file_utils.py

```diff
@@ -38,7 +38,11 @@ def copy_link(src, dst, update=0, verbose=1, dry_run=0, reproduce_link=False):
                  dir if os.path.basename(dst) == os.path.basename(src) else dst)
 
     if reproduce_link:
-        os.symlink(link_dest, dst, link_dest_isdir)
+        try:
+            os.symlink(link_dest, dst, link_dest_isdir)
+        except FileExistsError:
+            os.remove(dst)
```

Comment:
`os.unlink` please

Reply:
done

Reply:
Thanks. See my comment below though - I think it's safer to replicate the "copy" behavior - skip if exists. I would also prefer if a warning could be printed that file already exists.